Directory.Packages.props

```diff
@@ -228,4 +229,9 @@
     <PackageVersion Update="System.Formats.Asn1" Version="$(SystemFormatsAsn1Version)" />
     <PackageVersion Update="System.Text.Json" Version="$(SystemTextJsonVersion)" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.Bcl.Memory" Version="9.0.6" />
+    <PackageVersion Include="Microsoft.Bcl.TimeProvider" Version="9.0.6" />
```

Comment:
Can we also add these two via dependency flow so they get automatically updated? Let me know if you need help with that.